### PR TITLE
Add ToC, Overview, Audience and Prereqs sections and fix link in Billing section

### PR DESCRIPTION
**Pull request by gsanchezu**
@rchakra1, @dsimon1992, 👍?

### CODE REVIEW

File: Cloud Application Manager/Billing/billing-menu.md [hunk 1]
```diff
@@ -6,10 +6,32 @@
...
+### Prerequisites
+
+* Access to Cloud Application Manager, Management site.
+
+* The user must be an Administrator of the organization in Cloud Application Manager.
+
+* The user should be at the organization level scope to access the Billing option in the left side menu.
 
 ### Billing Dashboard
 When you click on **Billing** in the left side menu you will land into a dashboard page, where a bar graph with the billed amount for the latest months will be shown. You can also change the range of months being shown in the bar graph by selecting a different value from the to p-right dropdown on the graph.
```
Comment (dsimon1992) [inline Cloud Application Manager/Billing/billing-menu.md]: > You can also change the range of months being shown in the bar graph by selecting a different value from the **to p-right** dropdown on the graph.

Typo in bold

Comment (gsanchezu) [reply Cloud Application Manager/Billing/billing-menu.md]: 👍

File: Cloud Application Manager/Billing/billing-menu.md [hunk 2]
```diff
@@ -6,13 +6,35 @@
 "contentIsHTML": false
 }}}
 
+## Billing Menu
```
Comment (jim-phillips) [inline Cloud Application Manager/Billing/billing-menu.md]: `###` is the minimum

File: Cloud Application Manager/Billing/billing-separate-invoices.md [hunk 3]
```diff
@@ -6,17 +6,37 @@
 "contentIsHTML": false
 }}}
 
+## Billing in separate invoices
```
Comment (jim-phillips) [inline Cloud Application Manager/Billing/billing-separate-invoices.md]: `###` is the minimum

**Pull request merged**